tests/console/yast2_proxy.pm

```diff
@@ -80,22 +80,27 @@ sub run {
 
     # if firewall is enabled, then send_key alt-p, else move to page http ports
     unless ($is_still_susefirewall2) {
-        if (check_screen 'yast2_proxy_firewall_enabled') {
+        if (check_screen 'yast2_proxy_firewall_enabled', 10) {
             send_key 'alt-p';
             wait_still_screen 3;
+            # now we have a new popup to workaround for question about firewall settings
+            if (check_screen 'yast2_proxy_squid_cannot-open-interface', 3) {
```

Comment:
I don't think that "3" seconds is a good choice. It should be either 0 seconds when the previous "wait_still_screen" made sure that the dialog shows up correctly in this case or the default when we can not ensure anything. But the best approach should be to use a multi-tag `assert_screen` with `match_has_tag`. Can you point me to an openQA test that detected this screen and recorded the following soft-failure?